<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSpeedDialEntryDlg</class>
 <widget class="QWidget" name="EditSpeedDialEntryDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>280</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Name</string>
       </property>
       <property name="buddy">
        <cstring>nameWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameWidget">
       <property name="toolTip">
        <string>The name you want displayed on the Speed Dial button</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Url</string>
       </property>
       <property name="buddy">
        <cstring>urlWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KUrlRequester" name="urlWidget">
       <property name="mode">
        <set>KFile::Mode::ExistingOnly|KFile::Mode::File</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Icon</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="useFaviconBtn">
        <property name="text">
         <string>Use favicon</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="useLocalIconBtn">
          <property name="text">
           <string>Use local icon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KIconButton" name="localIconBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="iconSize">
           <number>48</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="useRemoteIconBtn">
          <property name="text">
           <string>Use remote icon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="remoteIconWidget">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kiconbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameWidget</tabstop>
  <tabstop>urlWidget</tabstop>
  <tabstop>useFaviconBtn</tabstop>
  <tabstop>useLocalIconBtn</tabstop>
  <tabstop>localIconBtn</tabstop>
  <tabstop>useRemoteIconBtn</tabstop>
  <tabstop>remoteIconWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
